wafflehaus/edit_response/__init__.py

```diff
@@ -52,27 +54,55 @@ def _replace_lookup(self, replace_str):
             return {}
         return replace_str
 
+    def _foreach(self, conditional, data):
+        if not isinstance(data, list):
+            return data
+        new_data = []
+        splits = conditional.split(":", 2)
+        action = splits[1]
+        conditionals = splits[2].split(',')
+        for item in data:
+            do_action = False
+            for cond in conditionals:
+                # TODO(jlh): Probably should support more than '='
+                (target, value) = cond.split("=")
+                if target not in item:
+                    continue
+                if item.get(target) == value:
```

Comment:
This requires item to be a dict, so you probably want an isinstance check before here. Otherwise if data is a list of strings, lists, etc., it'll raise.